Library/Homebrew/dev-cmd/livecheck.rb

```diff
@@ -51,6 +51,8 @@ def livecheck
     formulae_to_check = if args.tap
       Tap.fetch(args.tap).formula_names.map { |name| Formula[name] }
     elsif args.installed?
+      # Livecheck will carry out a normal check for formulae having `stable` and `head` that are
+      # installed as `head`, rather than a HEAD-only check.
```

Comment:
It may just be me but this doesn't feel like the best place for this comment. It would be nice to explain the idea behind this (i.e., livecheck will only use the installed `head` version for head-only formulae) but I don't think it makes sense here.

It may be better to explain this where the `current` variable is defined, as that's where the behavior comes into play (I'll add a comment for this below). I think that using the `stable` version for comparison except with head-only formulae is predictable and in-line with livecheck's normal behavior, so I don't think it's necessary to explain this in relation to the `--installed` switch.